src/app/clients/getExpandedClientDetails.ts

```diff
@@ -0,0 +1,191 @@
+            extra_information: "",
+        };
+    }
+
+    const client = rawClientDetails.client!;
+
+    return {
+        "voucher_#": rawClientDetails.voucher_number,
+        full_name: client.full_name,
+        phone_number: client.phone_number,
+        packing_date: formatDatetimeAsDate(rawClientDetails.packing_datetime),
+        packing_time: formatDatetimeAsTime(rawClientDetails.packing_datetime),
+        delivery_instructions: client.delivery_instructions,
+        address: formatAddressFromClientDetails(client),
+        household: formatHouseholdFromFamilyDetails(client.family),
+        "Age & Gender of Children": formatBreakdownOfChildrenFromFamilyDetails(client.family),
+        dietary_requirements: client.dietary_requirements.join(", "),
+        feminine_products: client.feminine_products.join(", "),
+        baby_products: client.baby_food,
+        pet_food: client.pet_food.join(", "),
+        other_requirements: client.other_items.join(", "),
+        extra_information: client.extra_information,
+    };
+};
+
+export const formatDatetimeAsTime = (datetime: string | null): string => {
+    if (datetime === null || isNaN(Date.parse(datetime))) {
+        return "-";
+    }
+
+    return new Date(datetime).toLocaleTimeString("en-GB");
+};
+
+export const formatAddressFromClientDetails = (
+    client: Pick<
+        Schema["clients"],
+        "address_1" | "address_2" | "address_town" | "address_county" | "address_postcode"
+    >
+): string => {
+    return [
+        client.address_1,
+        client.address_2,
+        client.address_town,
+        client.address_county,
+        client.address_postcode,
+    ]
+        .filter((field) => field)
+        .join(", ");
+};
+
+export const formatHouseholdFromFamilyDetails = (
+    family: Pick<Schema["families"], "age" | "gender">[]
+): string => {
+    let noAdults = 0;
+    let noChildren = 0;
+
+    for (const familyMember of family) {
+        if (familyMember.age === null || familyMember.age >= 18) {
+            noAdults++;
+        } else {
+            noChildren++;
+        }
+    }
+
+    const adultChildBreakdown = [];
+
+    if (noAdults > 0) {
+        adultChildBreakdown.push(`${noAdults} ${noAdults > 1 ? "adults" : "adult"}`);
+    }
+
+    if (noChildren > 0) {
+        adultChildBreakdown.push(`${noChildren} ${noChildren > 1 ? "children" : "child"}`);
+    }
+
+    const familyCategory = familyCountToFamilyCategory(family.length);
+    const occupantDisplay = `Occupant${noAdults + noChildren > 1 ? "s" : ""}`;
+
+    return `${familyCategory} ${occupantDisplay} (${adultChildBreakdown.join(", ")})`;
+};
+
+export const formatBreakdownOfChildrenFromFamilyDetails = (
+    family: Pick<Schema["families"], "age" | "gender">[]
+): string => {
+    const childDetails = [];
+
+    for (const familyMember of family) {
+        if (familyMember.age !== null && familyMember.age < 18) {
+            const age = familyMember.age === -1 ? "0-17" : familyMember.age;
```

Comment:
Small comment but age is currently a string if its -1 and a number otherwise. Can you add a toString so its consistent? Sorry this is just me being me T^T

Reply:
I've added a `.toString()` on `familyMember.age` to make the type of `age` consistent